relay-server/src/actors/global_config.rs

```diff
@@ -40,6 +40,23 @@ type UpstreamQueryResult =
 struct GetGlobalConfigResponse {
     #[serde(default)]
     global: Option<GlobalConfig>,
+    // Instead of using [`Status`], we use StatusResponse as a separate field in order to not
+    // make breaking changes to the api.
+    global_status: Option<StatusResponse>,
```

Comment:
Doesn't this also need a `#[serde(default)]` attribute (like `global` above)?

Reply:
huh id think that was necessary yeah, but it's weird integration tests didnt fail because of that, since minisentry don't return a global_status

Reply:
anyway i added it

Reply:
We could remove the top `#[serde(default)]`, serde implicitly has the default behaviour for Option's, they are special in that sense.